aiorospy/src/aiorospy/action.py

```diff
@@ -1,6 +1,13 @@
 import asyncio
+import janus
 
-from actionlib import SimpleActionClient, SimpleActionServer
+from actionlib import ActionClient
```

Comment:
Why?

Reply:
Learned it from another project and it seems cleaner to me!